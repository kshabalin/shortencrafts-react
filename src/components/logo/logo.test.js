import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallowToJson } from 'enzyme-to-json';

import Logo from './logo';

configure({ adapter: new Adapter() });

it('should render correctly', () => {
    const output = shallow(
        <Logo />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
});
