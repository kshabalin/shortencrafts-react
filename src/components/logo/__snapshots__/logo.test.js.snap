// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<img
  alt="shortencrafts"
  height="21"
  src="logo.png"
  width="191"
/>
`;
